example/client/BUILD

```diff
@@ -0,0 +1,11 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_binary")
+
+go_binary(
+  name = "client",
+  srcs = ["client.go"],
```

Comment:
Can we mark this BUILD target as "testonly = True"?

Similarly for the server-side API and for echo.go.